### PR TITLE
spec file: Add proper `Summary:` and `%description`

### PR DESCRIPTION
**Pull request by Olf0**
… taken from [SailfishOS:Chum Web PR 14](https://github.com/sailfishos-chum/sailfishos-chum-web/pull/4/files).

### CODE REVIEW

File: rpm/chum.spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Summary:        SailfishOS:Chum community repository
+Summary:        Proper SSU configuration for the SailfishOS:Chum community repository
```
Comment (rinigus) [inline rpm/chum.spec:1]: Don't think we need "Proper" in Summary. As if there are packages without "proper" SSU config.

File: rpm/chum.spec [hunk 2]
```diff
@@ -12,10 +12,19 @@ Conflicts:      sailfishos-chum-gui
...
+Note that the SailfishOS:Chum GUI application provides the same local
+repository configuration, while also providing a GUI app, which can be used in
+addition to pkcon or zypper. Furthermore it offers easy switching between the
+regular SailfishOS:Chum repository and the SailfishOS:Chum testing repository.
+Hence you might rather install the sailfishos-chum-gui RPM instead of the
+sailfishos-chum RPM.
 
 %package testing
-Summary:        SailfishOS:Chum community repository: Testing repository
+Summary:        Proper SSU configuration for the SailfishOS:Chum TESTING repository
```
Comment (rinigus) [inline rpm/chum.spec:27]: Same here

**Review by rinigus — CHANGES_REQUESTED** [review]

File: rpm/chum.spec (hunk 1 above)
Comment (Olf0) [reply rpm/chum.spec]: Absolutely agreed: Though I can easily imagine an "improper" SSU configuration (for a fixed SFOS release version or in the "Global" section), without some background knowledge this is impossible to understand.

File: rpm/chum.spec (hunk 2 above)
Comment (Olf0) [reply rpm/chum.spec]: See above.

**Review by rinigus — APPROVED** [review]

**Pull request merged**